pkg/glusterutils/healinfo_gd1.go

```diff
@@ -40,7 +40,7 @@ func (g *GD1) getHealDetails(cmd string) ([]HealEntry, error) {
 // HealInfo gets gluster vol heal info (GD1)
 func (g GD1) HealInfo(vol string) ([]HealEntry, error) {
 	// Get the overall heal count
-	cmd := fmt.Sprintf("vol heal %s info", vol)
+	cmd := fmt.Sprintf("vol heal %s events info | head -1000 | grep \"<gfid:\" | wc -l", vol)
```

Comment:
I believe this might work even though I haven't tested it, it ultimately calls the following method:

```
// execGluster runs `gluster` with --xml --remote-host=<...> and the args provided
func (g *GD1) execGluster(args ...string) ([]byte, error) {
```

Reply:
"events" is superfluous - the %s is the volume. But I think ultimately since this gets run with `--xml` it's not going to work anyway since it will have to parse the entire list to generate the XML.